### DISCUSSION TITLE
Where is the workflow saved?

### DISCUSSION THREAD

**Discussion by jjcruz404** [comfyanonymous/ComfyUI#5753] [Q&A] [answered]
Where is the workflow saved? And why can't it ask anymore where to save workflows in a directory. New UI is a headache. I looked at settings there's nothing there that can change it.

**Comment by LukeG89** [comment, marked as answer]
`ComfyUI\user\default\workflows`

Currently you can't choose a different folder.
But there is a Feature Request about it in the frontend repo:
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/688
